modules/configuring-external-databases/proc-configuring-default-configuration-for-secrets-and-pvcs.adoc

```diff
@@ -0,0 +1,84 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-default-configuration-for-secrets-and-pvcs_{context}"]
+= Configuring default configuration for Secrets and PVCs
+
+By default, the mount path is the {product-short} container's working directory, and if you do not define the mount path, it defaults to `/opt/app-root/src`. `app-config`, `configmap-files`, `secret-files`, `dynamic-plugins`, and PVCs mount to the container as either files or directories. The `default-config/deployment.yaml` file defines the dynamic plugins directory, as shown:
```

Comment:
As you can see here: https://github.com/redhat-developer/rhdh-operator/blob/main/docs/configuration.md
Supported objects for rhdh.redhat.com/mount-path are: pvcs, secret-files 
Supported objects for rhdh.redhat.com/containers: pvcs, secret-files, secret-envs
(Not `app-config`, `configmap-files` or `dynamic-plugins`)